config/zones/CA-SK.yaml

```
@@ -4,12 +4,16 @@ bounding_box:
   - - -100.86433964677866
     - 60.50005890629359
 capacity:
```

```suggestion
contributors:
  - nicolasbonifas
capacity:
```
Let's make sure you get the credit you deserve 😉